src/datachain/lib/dc.py

```diff
@@ -1069,13 +1069,22 @@ def mutate(self, **kwargs) -> "Self":
            Filename:    name(), parent(), file_stem(), file_ext()
            Array:       length(), sip_hash_64(), euclidean_distance(),
                         cosine_distance()
+           Window:      row_number(), rank(), dense_rank()
```

Comment:
I guess we'll need sum, avg and all those usual suspects as well? (can be a followup)

Reply:
Right! group_by functions needs to be mentioned as well.

Reply:
Ok, so we have group_by functions docs, it is in `group_by` chain method.
I have updated the docs for this: https://github.com/iterative/datachain/pull/515/files#diff-0b1cfd3d89c057cc57811e07eb20298eeaaee60c54cc66e5128f618d144f42eeR1002-R1021

Reply:
Also updated docs for all functions: https://github.com/iterative/datachain/pull/515/files#diff-c9b298e2d861e6d90972609a2bfcf0043b470ed946543dbf8bd1358cfafd1de7 + https://github.com/iterative/datachain/pull/515/files#diff-3abf137f75868839662512636c606626c4a44f52cafb75ca63b36d00ede0bb75R24-R53

Thank you for a reminder, this is very important 🙏